### DISCUSSION TITLE
本地部署无法进行对话：“无效的令牌” or “wrong api key”

### DISCUSSION THREAD

**Discussion by Liozhang** [GaiZhenbiao/ChuanhuChatGPT#996] [Q&A]
纯小白一个，我在本地服务器进行部署，测试使用的是两个转发的接口，在chatbox或chatall可以正常使用。
我在config.json文件中填入了openai_api_key和openai_api_base
正常启动，但是无法获得任何回答：

![屏幕截图 2023-12-18 151816](https://github.com/GaiZhenbiao/ChuanhuChatGPT/assets/44505221/64f703fc-28e5-438c-80b8-1df44933d7e0)

**Comment by GaiZhenbiao** [comment]
API Key错误，这是你使用的第三方API返回的，并非川虎Chat的问题。

**Reply by Liozhang** [reply]
我结束进程后通过
`echo $OPENAI_API_KEY`
得到的`key`和`config_json`文件中配置的不一致，设置到`config_json`的`openai_api_key`似乎没有起到作用，我通过
`export OPENAI_API_KEY='sk-XXX'`
`export OPENAI_API_BASE='https://XXXX'`
再重新启动服务后能通话成功了，可能是我之前某些操作导致的，我会再排查下原因，谢谢你的解答！
